pkg/telemetryquerylanguage/contexts/tqltraces/traces.go

```diff
@@ -29,21 +29,21 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/telemetryquerylanguage/tql"
 )
 
-type SpanTransformContext struct {
```

Comment:
This is definitely not Span, you may call it `Trace` if you want.

Reply:
Thats the old name, so wouldn't be a problem with change to `TransformContext`.